packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -36,7 +37,7 @@ function webpackConfigSingleSpa(opts) {
           }
         },
         {
-          test: /\.js$/,
+          test: /\.(js|ts)x?$/,
```

Comment:
The jest config should also be updated for this. See https://github.com/single-spa/create-single-spa/blob/master/packages/generator-single-spa/src/react/templates/jest.config.js

Reply:
Oh wait - disregard. The jest config already is allowing for `.tsx` files:

https://github.com/single-spa/create-single-spa/blob/7408db6ef7d30ecde2436a3c1a4653f0c51fad03/packages/generator-single-spa/src/react/templates/jest.config.js#L3